symengine/polynomial.h

```diff
@@ -204,10 +204,9 @@ RCP<const UnivariatePolynomial> mul_uni_poly(RCP<const UnivariatePolynomial> a,
                                              RCP<const UnivariatePolynomial> b);
 
 inline RCP<const UnivariatePolynomial>
-univariate_polynomial(RCP<const Symbol> i, unsigned int deg,
-                      const map_int_Expr &&dict)
+univariate_polynomial(RCP<const Symbol> i, map_int_Expr &&dict)
```

Comment:
`from_dict()` uses `erase` to remove member of dictionary with coefficients `0` . This will not be possible for a `const` dictionary